aws-k8s-toolbox/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM debian:bullseye-slim
```

Comment:
this lacks an entry in [here](https://github.com/mendersoftware/mender-test-containers/blob/master/.github/dependabot.yml)